### ISSUE TITLE
core(kb): support export and import a knowledge base via cli

### ISSUE THREAD

**Issue by sykp241095** [pingcap/autoflow#490]

**Comment by 634750802** [comment]
Related https://github.com/pingcap/autoflow/issues/398

**Comment by Mini256** [comment]
Do we really have such a scenario?

**Comment by sykp241095** [comment]
> Do we really have such a scenario?

Yes, sometimes when user uses a local and private network environment, it is difficult for them to download docs.pingcap.com or other online docs. This function can help them to download an existing knowledge base and import it to their own self-hosted autoflow easily.

**Comment by Mini256** [comment]
> help them to download an existing knowledge base

What would the existing knowledge base be, a internal website or a folder containing a lot of local files? Please provide a detailed description in the issue description.

If the data source is not common, we should use custom script to implement

>  import it to their own self-hosted autoflow

Why not using upload local file data source? Do we have to use CLI to upload?

**Comment by sykp241095** [comment]
> What would the existing knowledge base

For examples, TiDB knowledgebase, redis kb, mongodb kb.



> Why not using upload local file data source

* Cost:
If we add tidb knowledge by crawl docs.pingcap.com, users should pay again for llm while extract knowledge graphs from about thousand of pages;
if we achieve this by upload an about 100MB tidb-user-guide.pdf, it still need llm to extract the whole knowledge graph from this pdf file, it will cost about $50< cost <$100 maybe.

* LLM Performance
Users may not have smartest llm for knowledge graph extraction, for example many users use llama3.* 32B, or self-hosted model. these llm didn't have high performance for extracting and building graphs
